lib/unison-prelude/src/Unison/Prelude.hs

```diff
@@ -40,6 +40,7 @@ import Data.Either.Extra (eitherToMaybe, maybeToEither)
 import Data.Foldable as X (asum, fold, foldl', for_, toList, traverse_)
 import Data.Function as X ((&))
 import Data.Functor as X
+import Data.Functor.Identity as X
```

Comment:
You'll see a couple places where I had to remove duplicate imports of this from modules